web-console/4.0-designs/resource-quota/reqource-quota.md

```diff
@@ -0,0 +1,48 @@
+# Reqource Quota
+
+![](img/resource-quota-details.png)
+
+## Compute Resources
+![](img/compute-resources.png)
+- Compute resource request and limit utilization are displayed at the top of the overview as gauges
+
+## Scopes
+![](img/Scopes.png)
+- Each scope applied to the resource quota should be visible in the overview along with a short description below. Descriptions should be as follows:
+- Best Effort
+  - `Affects pods that do not have resource limits set. These pods have a best effort quality of service.``
+- Not Best Effort
+  - `Affects pods that have at least one resource limit set. These pods do not have a best effort quality of service.``
+- Terminating
+  - `Affects pods that have an active deadline. These pods usually include builds, deployers, and jobs.``
+- Not Terminating
+  - `Affects pods that do not have an active deadline. These pods usually include your applications.``
```

Comment:
Double back ticks `` for these descriptions are preventing them from rendering as code.